react-app/src/App.js

```diff
@@ -0,0 +1,31 @@
+import logo from './logo.svg';
+import './App.css';
+import {SecondHeader, ThirdHeader, FourthHeader} from './components/first-element';
+
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
You may remove from your app this snippet being the part of CRA boilerplate not to obscure the view and get your components into the plain view (thus, eliminate the need to scroll to the bottom of the page)